src/types/Patterns.ts

```diff
@@ -41,6 +41,7 @@ export const Patterns = {
   LABEL: /^[a-z0-9-_ ]{1,80}$/i,
   BPN: /^BPNL[0-9A-Z]{12}$/i,
   MAIL: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*))@(([a-z0-9-]+\.)+[a-z]{2,})$/,
+  PHONE: /^(\+|00)?(?:\d\s?){6,14}\d$/,
```

Comment:
this 7 digit check running on country code prefix also. So +14 12345 also true but which is an invalid case